examples/umami/admin.rs

```diff
@@ -0,0 +1,210 @@
+}
+
+/// Load and edit a random article.
+pub async fn edit_article(user: &GooseUser) -> GooseTaskResult {
+    // First, load a random article.
+    let nodes = common::get_nodes(&common::ContentType::Article);
+    let article = nodes.choose(&mut rand::thread_rng());
+    let goose = user.get(article.unwrap().url_en).await?;
+    common::validate_and_load_static_assets(user, goose, article.unwrap().title_en).await?;
+
+    // Next, load the edit link for the chosen article.
+    let mut goose = user
+        .get(&format!("/en/node/{}/edit", article.unwrap().nid))
+        .await?;
+
+    let mut saved_article;
+    match goose.response {
+        Ok(response) => {
+            // Copy the headers so we have them for logging if there are errors.
+            let headers = &response.headers().clone();
+            match response.text().await {
+                Ok(html) => {
+                    // Be sure we've properly loaded the edit page.
+                    let title = "Edit Article";
+                    if !common::valid_title(&html, title) {
+                        return user.set_failure(
+                            &format!("{}: title not found: {}", &goose.request.url, title),
+                            &mut goose.request,
+                            Some(&headers),
+                            Some(&html),
+                        );
+                    }
+
+                    // Load all static elements on the page, as a real user would.
+                    common::load_static_elements(user, &html).await;
+
+                    // Scrape the HTML to get the values needed in order to POST to the
+                    // log in form.
+                    let form_build_id = common::get_form_value(&html, "form_build_id");
+                    if form_build_id.is_none() {
+                        return user.set_failure(
+                            &format!("{}: no form_build_id on page", goose.request.url),
+                            &mut goose.request,
+                            Some(&headers),
+                            Some(&html),
+                        );
+                    }
+                    let form_token = common::get_form_value(&html, "form_token");
+                    if form_token.is_none() {
+                        return user.set_failure(
+                            &format!("{}: no form_token on page", goose.request.url),
+                            &mut goose.request,
+                            Some(&headers),
+                            Some(&html),
+                        );
+                    }
+
+                    // Build node form with random word from title.
+                    let params = [
+                        ("form_build_id", &form_build_id.unwrap()),
+                        ("form_token", &form_token.unwrap()),
+                        ("form_id", &"node_article_edit_form".to_string()),
+                        ("op", &"Save (this translation)".to_string()),
```

Comment:
@LionsAd Is this sufficient to trigger a D9 cache flush? The logged-in user simply loads the edit page and submits the node edit form.